charts/rbs/values.yaml

```diff
@@ -27,9 +27,11 @@ vault:
   enabled: true
   # vault.pathPrefix -- (string) Vault path prefix for secrets. Required if vault.enabled.
   pathPrefix:
-  # vault.adminUserFilePath -- (string) Vault path prefix for admin user list. Required if vault.enabled.
-  # Use the same users as admin for all env by default. Override if needed in helmfile repo.
-  adminUserFilePath: config/terra/rbs/common/iam
+  # vault.janitorConfigPath -- (string) Janitor related configs in Vault path prefix. Required if vault.enabled.
+  # Now those Janitor configs are generated by Terraform and output to Vault.
```

Comment:
let's improve this comment. I think this is also required if cleanupAfterHandout is true. Can we be more specific than `Janitor related configs`? Maybe this ssould be janitorPubSubConfigPath?